e2e/jest.test-setup.ts

```diff
@@ -186,33 +194,34 @@ beforeEach(async () => {
   });
 
   page.on('console', async (message) => {
-    if (!message.args().length) {
-      return;
-    }
-    const title = await getTitle();
+    if (!message.args().length) return;
+    const title = await getPageTitle();
     try {
       const args = await Promise.all(message.args().map(a => describeJsHandle(a)));
-      console[message.type() === 'warning' ? 'warn' : message.type()](`❗ ${title}\n${message.text()}`, ...args);
+      console[message.type() === 'warning' ? 'warn' : message.type()](`Page console: "${title}"\n`, ...args);
+      console.log('');
     } catch (err) {
-      console.error(`❗ ${title}\nException occurred when getting console message.\n${err}\n${message.text()}`);
+      console.error(`❗ "${title}"\nException occurred when getting page console message.\n${err}\n${message.text()}`);
     }
   });
 
   page.on('error', async (error) => {
-    const title = await getTitle();
+    const title = await getPageTitle();
     try {
-      console.error(`❗ ${title}\nError message: ${error.message}\nStack: ${error.stack}`);
+      console.error(`PAGE ERROR: "${title}"\n${error.toString()}\n${error.message}\n${error.stack}`);
+      console.log('');
     } catch (err) {
-      console.error(`❗ ${title}\nException occurred when getting error.\n${err}`);
+      console.error(`❗ "${title}"\nException occurred when getting page error.\n${err}`);
     }
   });
 
   page.on('pageerror', async (error) => {
```

Comment:
FYI: Somehow logging of console `pageerror` and `error` are not working. I don't have a solution at this time.
```
console.error
      PAGEERROR: "Edit Workspace | [Test] All of Us Researcher Workbench"
      Error: Object
      Object

      219 |     const title = await getPageTitle();
      220 |     try {
    > 221 |       console.error(`PAGEERROR: "${title}"\n${error.toString()}\n${error.message}\n${error.stack}`);
          |               ^
      222 |       console.log('');
      223 |     } catch (err) {
      224 |       console.error(`❗ "${title}"\nPage exception occurred when getting pageerror.\n${err}`);

      at jest.test-setup.ts:221:15
   ```